_haikus/its_everywhere.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Its Everywhere
```

Comment:
In this context, "Its" should be "It's" for "it is" contraction rather than possessive.